test/recipes/20-test_enc.t

```diff
@@ -26,6 +26,8 @@ my $testsrc = srctop_file("test","recipes",basename($0));
 my $test = catfile(".", "p");
 
 my $cmd = "openssl";
+my $bldtop = bldtop_dir("providers");
+my @prov = ("-provider_path", $bldtop, "-provider", "default", "-provider", "legacy");
```

Comment:
That's an odd name for a subdirectory of the build top. How about `$providers`?

Reply:
Changed all 3 files to use $provpath